### DISCUSSION TITLE
Multiple groups and homes

### DISCUSSION THREAD

**Discussion by RickRiet** [andrei-tatar/node-red-contrib-smartnora#222] [General]
He,

I have a issue with multiple Instances of Node-Red and multiple Houses and google accounts, is there a way to prevent devices in 1 Group to be added to a house.
Secondly the other google account now gets erros that smart nora is not available is there a fix for that?

**Comment by andrei-tatar** [comment]
Hi @RickRiet,
What's your user id?

**Comment by RickRiet** [comment]
My user id is: uF4HL9LH7CQVQOnqakf3VAYb65H2

**Reply by andrei-tatar** [reply]
@RickRiet I would update the package to the latest version. I see you're using 1.15. Latest is 1.17
As long as you're using separate group names for each node-red instance, you should be good. You have 2 groups defined right now.

As far as houses go, I think you can manage that from the Google Home app. Create a new home and move devices between them:
1. Click on the device in Google Home
2. Click on the settings icon (gear shaped)
3. Click on home and select `Add another home`
4. etc.

Once they are all setup, they should stay in the new home.

There is a `Home name` in the nora config that is passed to google as `Structure hint` which is supposed to be a hint on the name of the home the device belongs to but I've never seen it working on Google's end (Home is not automatically created and devices assigned to it).

https://developers.google.com/assistant/smarthome/reference/rest/v1/devices/sync
`structureHint`
> Suggested name for the structure where this device is installed. Google attempts to use this value during user setup.

**Reply by RickRiet** [reply]
He andrei,

Thanks for the answer i got multiple houses but since the structurehint is not being followed it need to move the devices everytime i make a mistake and devices are disconnected from smart-nora.

I will let you know if the update helps.

Rick

**Reply by RickRiet** [reply]
He Andrei, updateing to the newest version didn't work google still responds with can't reach smart nora in 1 of the houses.

**Reply by andrei-tatar** [reply]
@RickRiet any logs in node-red?

**Reply by RickRiet** [reply]
HE Andrei,

No it seems to be fully on the connection to smart-nora cloud. Local excecution is working flawelessly even though i get the error from google assistant.

**Reply by andrei-tatar** [reply]
@RickRiet What error are you getting from the assistant?

**Reply by RickRiet** [reply]
![image](https://user-images.githubusercontent.com/76628445/202140878-8d65092c-0b95-4bb8-8ed1-77f81385cb64.jpeg)

**Reply by RickRiet** [reply]
He andrei,

It magically fixed itself, thanks for your help!

**Reply by andrei-tatar** [reply]
@RickRiet Must've been a problem at Google
